scalability_and_performance/recommended-cluster-scaling-practices.adoc

```diff
@@ -5,6 +5,14 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+
+[IMPORTANT]
+====
+The guidance in this section is only relevant for installations with cloud
```

Comment:
@ahardin-rh The recommendations are definitely specific to public cloud providers and are not applicable to bare metal installs. We can remove the following line as it might be misleading like @smalleni mentioned: "Currently, there is no MachineSet available with bare metal installations."

Reply:
@chaitanyaenr Thanks! Updated.